internal/pkg/describe/rd_web_service.go

```diff
@@ -55,6 +54,15 @@ func NewRDWebServiceDescriber(opt NewServiceConfig) (*RDWebServiceDescriber, err
 	return describer, nil
 }
 
+// ServiceARN retrieves the ARN of the app runner service.
+func (d *RDWebServiceDescriber) ServiceARN(env string) (string, error) {
+	err := d.initClients(env)
+	if err != nil {
+		return "", err
+	}
+	return d.envSvcDescribers[env].ServiceARN()
```

Comment:
probably should check and make sure their is a describer for `env` before calling the function so we don't panic
```
ard, ok := d.envSvcDescriber[env]
if !ok || ard == nil {
    // error
}

return ard.ServiceARN()
```

Reply:
Good call! Addressed